documentation/CAMARA-API-Design-Guide.md

```diff
@@ -592,13 +594,22 @@ info:
 ```
 
 #### 5.3.1. Title
-Title describes the API shortly. The title shall not include the term "API" in it.
+`title` field describes the API shortly. The title shall not include the term "API" in it.
 
 #### 5.3.2. Description
-No special restrictions specified in CAMARA.
+`description`field: There are no special restrictions specified in CAMARA for the documentation explaining API.
+[CommonMark syntax](https://spec.commonmark.org/) may be used for rich text representation.
+Images hosted in Github API repository can be embedded in the description, for example:

```

Comment:
Well, images NOT in the repository may change at any time. 

```suggestion
It is not recommended to link images outsite of the Github API repository, since changes to these images may be outside of control of repository maintainers. Images should be preferably hosted within the API repository in the matching release branch.

```